code/modules/clothing/glasses/glasses.dm

```diff
@@ -129,6 +129,54 @@
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 	glass_colour_type = /datum/client_colour/glass_colour/green
 
+/obj/item/clothing/glasses/night/mining
+	name = "Planetary Night Vision Goggles"

```

Comment:
none of the other things are but probably _technically_ yes
I don't feel like updating all of them though